lib/util/npm.js

```diff
@@ -5,10 +5,14 @@ module.exports = (npmBin, npmCommand, cwd, env, extra) => {
   const isJS = npmBin.endsWith('.js')
   const cmd = isJS ? process.execPath : npmBin
   const args = (isJS ? [npmBin] : []).concat(npmCommand)
+
+  // Avoid "spawn npm ENOENT" on Windows by setting { shell: true }
+  const shell = (process.platform === 'win32' && npmBin === 'npm') ? true : undefined
```

Comment:
I think this is actually a more broad issue than with just this repo.  Historically we've omitted `shell: true` for security reasons.

You can read some of the prior work being done to triage and fix that particular bug in this PR: https://github.com/npm/promise-spawn/pull/118

Reply:
Reverted this